evergreen.yml

```diff
@@ -1685,6 +1744,8 @@ tasks:
       - func: "install rust toolchain"
       - func: set and check packages version
       - func: "generate SBOM and scan"
```

Comment:
Let's split up the generate and scan into 2 different parts.
This way we can push and pull an augmented SBOM even if scanning fail